server/sonar-web/src/main/js/apps/tutorials/onboarding/Onboarding.js

```diff
@@ -90,6 +93,8 @@ export default class Onboarding extends React.PureComponent {
           open={step === 'token'}
           stepNumber={organizationsEnabled ? 2 : 1}
         />
+
+        <AnalysisStep open={step === 'analysis'} stepNumber={organizationsEnabled ? 3 : 2} />
```

Comment:
Maybe you could define a `stepNumber` variable initialized at 1 and then do `stepNumber={stepNumber++}` for every steps. So you would always have the correct numbers without doing tests depending on the orgs.